docs/surfaces/TRANSPORT.md

```
@@ -226,6 +226,58 @@ curl -H "Authorization: Bearer $LAB_MCP_HTTP_TOKEN" \
+
+Gateway admin actions (`/v1/gateway/*`, `labby mcp` → `gateway` tool) manage the upstream MCP
+server registry.  **They can spawn arbitrary local stdio commands with labby's full process
+environment.**  The implications are non-negotiable:
+
+### HTTP surface (`labby serve`)
+
+1. **`/v1/gateway` is never mounted when auth is not configured.**
+   `api/router.rs` refuses to mount the gateway route group when neither a static bearer token
+   (`LAB_MCP_HTTP_TOKEN`) nor OAuth state (`LAB_AUTH_MODE=oauth`) is configured.  The startup
+   warning emitted is:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)